app/views/tasks/_form.html.erb

```diff
@@ -1,9 +1,13 @@
-<%= form_with(model: @task, local: true) do |f|%>
+<%= form_with(model: task, local: true) do |f|%>
   <%= f.label :title %>
   <%= f.text_field :title, class: 'form-control' %>
   <%= f.label :description %>
   <%= f.text_field :description, class: 'form-control' %>
   <%= f.label :date_and_time %>
   <%= f.datetime_local_field :date_and_time, class: 'form-control' %>
+  <% if !task.id.nil? %>
```

Comment:
`nil` in ruby is a 'falsy' value, so if you `if nil` is the same as `if false` the same with a not null value, like `if 1` or `if 'string'` is the same as `if true`.

This way you can make your code more simple with `unless task.id` or `if !task.id`